enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -37,8 +38,14 @@
      (sql.qp/->honeysql (or driver/*driver* :h2) (Table (mt/id table-key)))))
 
   ([table-key field-key]
-   (mt/with-everything-store
-     (sql.qp/->honeysql (or driver/*driver* :h2) (Field (mt/id table-key field-key))))))
+   (let [field-id   (mt/id table-key field-key)
+         field-name (db/select-one-field :name Field :id field-id)]
+     (mt/with-everything-store
+       (sql.qp/->honeysql
+        (or driver/*driver* :h2)
+        [:field field-id {::add/source-table  (mt/id table-key)
+                          ::add/source-alias  field-name
+                          ::add/desired-alias field-name}])))))
```

Comment:
This is a utlity function that takes a `Field` instance and compiles it to SQL by using the SQL QP for purposes of generating some test queries. Since the SQL QP no longer calculates appropriate aliases for stuff itself it was necessary to add that information in ourselves.